synapse/storage/databases/main/room.py

```diff
@@ -1396,6 +1480,9 @@ async def maybe_store_room_on_outlier_membership(
             insertion_values={
                 "room_version": room_version.identifier,
                 "is_public": False,
+                # We don't worry about setting the `creator` here because
+                # we don't process any messages in a room while a user is
+                # invited (only after the join).
                 "creator": "",
```

Comment:
> Do you need to worry about it? Depends what you want to use the field for, I guess :)

I think we do need to consider it. I've asked for specific clarification on [what I don't know there](https://github.com/matrix-org/synapse/pull/10697#discussion_r698924269).

---

> > Should this be `None` over `""`?
> 
> :man_shrugging: Maybe, but I'm not sure I'd bother changing it.

Leaving it as-is ⏩ Just need to consider it could be an empty string in the queries